docs/docs/references/users/README.md

```diff
@@ -36,7 +36,7 @@ which means you can query them using Management API.
 
 :::note
 
-- **Unique**: Ensures the _uniqueness_ of the values entered into an attribute of a DB table.
+- **Unique**: Ensures the [uniqueness](https://en.wikipedia.org/wiki/Unique_key#:~:text=uniqueness%20constraint) of the values entered into an attribute of a DB table.
```

Comment:
```suggestion
- **Unique**: Ensures the [uniqueness](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-UNIQUE-CONSTRAINTS) of the values entered into an attribute of a database table.
```

1. `#:~:text=uniqueness%20constraint` is a browser feature and unstable
2. since we're using Postgres, just link the Postgres docs is fine